website/docs/r/securityhub_custom_action.markdown

```diff
@@ -0,0 +1,48 @@
+---
```

Comment:
This page is missing a sidebar link in `website/aws.erb` -- the CI testing on master now catches this. 👍